crates/binjs_io/src/escaped_wtf8.rs

```diff
@@ -277,5 +283,176 @@ pub fn for_print(s: &SharedString) -> SharedString {
         };
     }
 
-    SharedString::from_string(String::from_utf8(buf).expect("Escaped string should be valid UTF-8"))
+    String::from_utf8(buf).expect("Unescaped string should be valid UTF-8")
+}
+
+/// If the given `bytes` contains any unicode escape \uXXXX and that is lone
```

Comment:
It's not exactly clear to me what happens if there are multiple escapes, e.g. `"\\\\uXXXX"`. Could you perhaps write this as a doctest?